Parsers/ASimUserManagement/Parsers/ASimUserManagement.yaml

```diff
@@ -0,0 +1,37 @@
+Parser:
```

Comment:
can we add imUserManagement.yaml too?

Also, we have another parser for usermanagement. can we add this to union parser? https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/ASimUserManagement/Parsers/ASimUserManagementCiscoISE.yaml

Reply:
Done

Reply:
changes not visible, please validate

Reply:
Done